config/packages/fos_elastica.yaml

```diff
@@ -152,6 +152,11 @@ fos_elastica:
                     property_path: false
                     type: 'date'
                     format: 'yyyy-MM-dd'
+                funders:
+                    type: 'nested'
+                    properties:
+                        id: 
```

Comment:
Do we need to index name as well?

Reply:
No, we don't need to index name, because we are searching based on `id`